.travis.yml

```diff
@@ -8,20 +8,121 @@ env:
+          name: "Jest unit tests"
+          script:
+              - npm run test -- $SCRIPT_ARGS
+        - name: "Web Automated Tests"
+          fast_finish: true
+          dist: trusty
+          addons:
+              chrome: stable
+          services:
+              - xvfb
+          before_install:
+              - export CHROME_BIN=chromium-browser
+              - export DISPLAY=:99.0
+              - sh -e /etc/init.d/xvfb start
+              - sleep 3
+          script:
+              - if [ "$WEB_PROJECTS" == "true" ] && [ "$PARTIAL_BUILD" == "true" ]; then travis_wait 30 npm run test:e2e -- $SCRIPT_ARGS; else echo "No individuals web package changed. Skipping tests"; fi
+        - name: "iOS Automated tests"
+          language: objective-c
+          os: osx
+          osx_image: xcode11.2
+          env:
+              - REACT_NATIVE_VERSION=0.59.9
+              - PATH=$PATH:~/Library/Python/2.7/bin
+              - CODE_SIGNING_REQUIRED=NO
+          before_install:
+              - nvm install node
+              - node -v
+              - npm -v
+              - npm install -g detox-cli
+              - gem install xcpretty
+              - brew tap wix/brew
+              - brew install applesimutils --HEAD
+              - cd detox
+          install:
+              - npm ci
+          script:
+              - if [ "$NATIVE_PROJECTS" == "true" ]; then npm run test:ios; else echo "No native package changed. Skipping ios tests"; fi
+        - name: "Android Automated tests"
+          os: osx
+          osx_image: xcode11.2
+          language: objective-c
+          env:
+              - PATH=$PATH:~/Library/Python/2.7/bin
+              - ANDROID_SDK_ROOT="/usr/local/share/android-sdk"
+          before_install:
+              - brew cask uninstall java
+              - brew cask install homebrew/cask-versions/adoptopenjdk8
+              - nvm install node
+              - node -v
+              - npm -v
+              - brew tap wix/brew
+              - mkdir ~/.android
+              - echo 'count=0' > ~/.android/repositories.cfg
+              - brew cask install android-sdk
+              - cd detox
+          install:
+              - npm ci
+          before_script:
+              - yes | sdkmanager --licenses >/dev/null
+              - echo y | sdkmanager "platform-tools" >/dev/null
+              - echo y | sdkmanager "tools" >/dev/null
+              - echo y | sdkmanager "build-tools;28.0.3" >/dev/null
+              - echo y | sdkmanager "platforms;android-28" >/dev/null
+              - echo y | sdkmanager "platforms;android-25" >/dev/null
+              - echo y | sdkmanager "emulator" >/dev/null
+              - echo y | sdkmanager "extras;android;m2repository" >/dev/null
+              - echo y | sdkmanager "system-images;android-25;google_apis;armeabi-v7a" >/dev/null
+              - avdmanager -s create avd -n EMULATOR -k 'system-images;android-25;google_apis;armeabi-v7a' -f -d 'pixel_xl'
```

Comment:
With osx could we not run on higher/different android version?

Reply:
We cant. Unfortunately emulator without hardware acceleration (ARM processor) just have till Android-25